test/orange/univ/SimpleUnitTracker.test.cc

```diff
@@ -760,7 +761,7 @@ TEST_F(FieldLayersTest, TEST_IF_CELERITAS_DOUBLE(heuristic_init))
         EXPECT_SOFT_EQ(0, result.failed);
     }
 
-    if (CELER_USE_DEVICE)
+    if (celeritas::device())
```

Comment:
This is slightly dangerous because a low-level bug that keeps the device from initializing means that this may pass.